### ISSUE TITLE
kann links nicht farblich verändern

### ISSUE THREAD

**Issue by alteroppa** [ca057/KITplayground#1]
meine style.css sieht momentan so aus:

```
p {
    font-family: "Verdana";
    font-size: 20px;
    color: #293d3d;
}

body {
    background-color: #84aeae;
}


a:link {
    color: green;
}



a:hover {
    color: black;
}


h3 {
    font-size: 30px;
    font-family: "Verdana";
    color: white;
}



@import(../bootstrap-four/style.css)}
```

Problematischerweise ändert a:link (color:green) nicht alle links, sondern nur einige... und ich weiß nicht, weshalb. Die Links sind teilweise unterschiedlichen Klassen zugeordnet, muss ich die alle ändern?
jmd ne Idee?

**Comment by ca057** [comment]
Was für ein Verhalten willst du denn erreichen? Vergebe am Besten mal alle Pseudo-Klassen, die für Links möglich sind (siehe [w3schools](http://www.w3schools.com/css/css_pseudo_classes.asp)). Dabei sind die folgenden Regeln zu beachten:
- a:hover MUST come after a:link and a:visited
- a:active MUST come after a:hover

Ansonsten kann es sein, dass durch `a:link {...}` nur die unbesuchten Links anders dargestellt werden, alle besuchten in der default-Farbe.

Weitere Fehlerquelle kann sein, dass durch den `@import(...)` die von dir festgelegten Styles wieder überschrieben werden. Sobald es eine speziellere oder später vorkommende Regel gibt, wird diese angewandt. Schiebe den import-Befehl am Besten nach ganz oben.

Genauer hab ich jetzt aber nicht reingeschaut, das fällt mir dazu nur gerade ein.

**Comment by ca057** [comment]
Oh, und was mir gerade noch auffällt: Wenn alle Texte auf der Seite Verdana als Font haben sollen, würde ich Verdana möglichst an erster Stelle z.B. im `body` definieren. Ebenso wenn die meisten Elemente das haben, dann überschreibt man es lieber für die Sonderfälle (oder schließt einzelne Klassen/Selektoren mit `:not(...)` aus &rarr; `:not()` aber nicht auf den `body` anwenden, das könnte zu komischen Nebeneffekten führen, sowas dann doch lieber nur mit Klassen machen). Und am Besten noch `, sans-serif` als default dahinterpacken!

Jetzt höre ich auf zu klugscheißen :D

**Comment by alteroppa** [comment]
nein, das passt voll mit dem klugscheißen, ich lerne mich eh erst ein ;) insofern bin ich dankbar für alle tipps... danke! Werds mir anschauen.
